routes/api/v1/meals.js

```diff
@@ -15,4 +23,28 @@ router.get("/", function(req, res, next) {
   })
 });
 
+router.get("/:id", function(req, res, next) {
+  res.setHeader("Content-Type", "application/json");
+  return meals.findByPk((req.params.id), {
+    attributes: ['id', 'name'],
+    include: [{
+      model: models.Food,
+      as: 'foods',
+      attributes: ['id', 'name', 'calories'],
+      through: {
+        attributes: []
+      }
+    }]
+  }).then(results => {
+    if (results) {
+      res.status(200).send(results)
+    } else {
+      res.status(404).send({message: "Meal cannot be found"})
+    }
+  })
+  .catch(error => {
+    res.status(500).send({error})
```

Comment:
Maybe we don't want to send the entire error back?  I remember Ian/Cory mentioning how this can lead to too much information (potentially compromising) being show to a user.